hypervisor/common/hypercall.c

```diff
@@ -199,6 +199,8 @@ int32_t hcall_create_vm(struct acrn_vm *vm, uint64_t param)
 					ret = copy_to_gpa(vm, &cv, param, sizeof(cv));
 				}
 			}
+		} else {
```

Comment:
Want to control the use of pr_err for LOC. This one may be not needed since the caller can easily identify it.

Reply:
OK, will remove it.